packages/bonde-styleguide/src/content/SwitchSlider/SwitchSlider.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import PropTypes from 'prop-types'
```

Comment:
destructuring PropTypes import

```suggestion
import { bool, func } from 'prop-types'
```